start-tests.sh

```diff
@@ -169,6 +169,8 @@ elif [[ ! -z $USE_HUB_KUBECONFIG && "$USE_HUB_KUBCONFIG" == true ]]; then
   ADD_KUBECONFIG="--kubeconfig=$OPTIONS_HUB_KUBECONFIG"
 fi
 
+echo -e "Logged in as user: $(oc whoami)\n"
```

Comment:
Why do we have this logging statement? Was this only used for developing the tests, or will it be used when debugging failures? I don't see the output being used anywhere.

Reply:
This was used for debugging purposes.

Reply:
I'd say it's fine to leave in until this test isn't in active development anymore